src/models/PrimaryAuth.js

```diff
@@ -37,9 +37,24 @@ function (Okta, BaseLoginModel, CookieUtil, Enums) {
       }
 
       return {
-        username: ['string', true, username],
+        username: {
+          type: 'string',
+          validate: function (value) {
+            if(!value || value.length < 1) {
+              return 'Please enter a username';
```

Comment:
If there is no way to just customize message to get to that use same validate function like courage.  /cc @ukilon-okta 
    if (field.required && (!isDefined || _.isNull(value) || value === '')) {